lib/swordappv2/LICENCE

```diff
@@ -0,0 +1,33 @@
+Copyright (c) 2009 - 2011, Stuart Lewis
```

Comment:
Just add the swordappv2 library as a submodule rather than committing it directly to the plugin repository.